[2021-11-19 10:42:16,564] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: file_upload.get_file scheduled__2021-11-08T00:00:00+00:00 [queued]>
[2021-11-19 10:42:16,614] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: file_upload.get_file scheduled__2021-11-08T00:00:00+00:00 [queued]>
[2021-11-19 10:42:16,614] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-19 10:42:16,614] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-11-19 10:42:16,616] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-19 10:42:16,714] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_file> on 2021-11-08 00:00:00+00:00
[2021-11-19 10:42:16,735] {standard_task_runner.py:52} INFO - Started process 13641 to run task
[2021-11-19 10:42:16,762] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'file_upload', 'get_file', 'scheduled__2021-11-08T00:00:00+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/file_loadin.py', '--cfg-path', '/tmp/tmpo5ol853m', '--error-file', '/tmp/tmpm0zy2mcv']
[2021-11-19 10:42:16,782] {standard_task_runner.py:77} INFO - Job 72: Subtask get_file
[2021-11-19 10:42:16,985] {logging_mixin.py:109} INFO - Running <TaskInstance: file_upload.get_file scheduled__2021-11-08T00:00:00+00:00 [running]> on host user-Latitude-3410
[2021-11-19 10:42:17,136] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=file_upload
AIRFLOW_CTX_TASK_ID=get_file
AIRFLOW_CTX_EXECUTION_DATE=2021-11-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-08T00:00:00+00:00
[2021-11-19 10:42:17,250] {transport.py:1819} INFO - Connected (version 2.0, client OpenSSH_for_Windows_7.7)
[2021-11-19 10:42:17,576] {transport.py:1819} INFO - Authentication (publickey) failed.
[2021-11-19 10:42:17,881] {transport.py:1819} INFO - Authentication (publickey) failed.
[2021-11-19 10:42:18,006] {transport.py:1819} INFO - Authentication (password) successful!
[2021-11-19 10:42:18,148] {sftp.py:158} INFO - [chan 0] Opened sftp connection (server version 3)
[2021-11-19 10:42:20,855] {logging_mixin.py:109} INFO -      Unnamed: 0 SALESDISCOUNT Sales Line Discount  ...  Unnamed: 10 Unnamed: 12               timestamp
0           0.0           NaN                 NaN  ...          NaN         NaN 2021-11-19 10:39:24.229
1           1.0          Type                Code  ...  Ending Date         NaN 2021-11-19 10:39:24.229
2           2.0          Item       FG-0000000001  ...   30/07/2020         NaN 2021-11-19 10:39:24.229
3           3.0          Item       FG-0000000002  ...   30/07/2020         NaN 2021-11-19 10:39:24.229
4           4.0          Item       FG-0000000003  ...   30/07/2020         NaN 2021-11-19 10:39:24.229
..          ...           ...                 ...  ...          ...         ...                     ...
155        27.0           NaN           MG-111125  ...          NaN         NaN 2021-11-19 10:41:26.551
156        28.0           NaN           MG-111126  ...          NaN         NaN 2021-11-19 10:41:26.551
157        29.0           NaN           MG-111127  ...          NaN         NaN 2021-11-19 10:41:26.551
158         NaN          Item           MG-111130  ...          NaN         NaN 2021-11-19 10:41:26.551
159         NaN          data           MG-111130  ...          NaN         NaN 2021-11-19 10:41:26.551

[160 rows x 14 columns] <class 'pandas.core.frame.DataFrame'>
[2021-11-19 10:42:21,566] {sftp.py:158} INFO - [chan 0] sftp session closed.
[2021-11-19 10:42:21,568] {python.py:152} INFO - Done. Returned value was: None
[2021-11-19 10:42:21,635] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=file_upload, task_id=get_file, execution_date=20211108T000000, start_date=20211119T051216, end_date=20211119T051221
[2021-11-19 10:42:21,735] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-19 10:42:21,795] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
